aws/resource_aws_s3_bucket.go

```diff
@@ -2585,7 +2585,7 @@ func rulesHash(v interface{}) int {
 	if v, ok := m["filter"].([]interface{}); ok && len(v) > 0 && v[0] != nil {
 		buf.WriteString(fmt.Sprintf("%d-", replicationRuleFilterHash(v[0])))
 
-		if v, ok := m["delete_marker_replication_status"]; ok && v.(string) == s3.DeleteMarkerReplicationStatusEnabled {
+		if v, ok := m["delete_marker_replication"]; ok && v.(string) == s3.DeleteMarkerReplicationStatusEnabled {
```

Comment:
should this stay as `delete_marker_replication_status` to match resource's schema?

Reply:
This was updated as part of the conversion to be "better aligned with the Go SDK".  In the s3 bucket resource it is a boolean, in here we want it to be an object with `status = "Enabled'`.  In doing that we are changing the argument name to be `delete_marker_replication`.   Having "status" in the argument name felt redundant.

Reply:
Oh, I see.  You are correct.  In the s3 bucket we want that to remain as is.  That looks like a mistake.